EmotiBitDataParser/bin/EmotiBitDataParser.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+exePath=$(pwd)
+dataDir=$(pwd)
+getHelp=false
+
+echo -e "";
+echo -e "** EmotiBitDataParser.sh ** ";
+echo -e "Runs EmotiBitDataParser on *.csv in passed data directory\n";
+echo -e 'Example input:\n./EmotiBitDataParser.sh -x "C:\\Program Files\\EmotiBit\\EmotiBit DataParser\\EmotiBitDataParser.exe" -d "C:\\priv\\local\\EmotiBitData\\"\n'
+
+while getopts x:d: flag
+do
+    case "${flag}" in
+        x) exePath=${OPTARG};;
+        d) dataDir=${OPTARG};;
+    esac
+done
+
+# Replace backslashes with forward slashes
+exePath="${exePath//\\//}"
+dataDir="${dataDir//\\//}"
+
+echo "[-x] exePath: $exePath";
+echo "[-d] dataDir: $dataDir";
+
+currDir=$(pwd);
+echo -e "\npwd: $currDir"
+
+for file in $(eval "ls *.csv")
```

Comment:
This needs a fix. Currently, it only looks for csv files in the dir the shell script is located in